skabase/SKALogger/test/SKALogger_test.py

```diff
@@ -15,33 +15,38 @@
 sys.path.insert(0, os.path.abspath(path))
 
 # Imports
-import pytest
 from mock import MagicMock
-
-from PyTango import DevState
-
-# PROTECTED REGION ID(SKALogger.test_additional_imports) ENABLED START #
-# PROTECTED REGION END #    //  SKALogger.test_additional_imports
+from PyTango import DevState, DeviceProxy
+import re
+import pytest
+import logging
+import calendar
+import time
+#from testfixtures import LogCapture
+logging.basicConfig()
+from logging.handlers import SysLogHandler
+
+# Note:
+#
+# Since the device uses an inner thread, it is necessary to
+# wait during the tests in order the let the device update itself.
+# Hence, the sleep calls have to be secured enough not to produce
+# any inconsistent behavior. However, the unittests need to run fast.
+# Here, we use a factor 3 between the read period and the sleep calls.
+#
+# Look at devicetest examples for more advanced testing
 
 
 # Device test case
 @pytest.mark.usefixtures("tango_context", "initialize_device")
-# PROTECTED REGION ID(SKALogger.test_SKALogger_decorators) ENABLED START #
-# PROTECTED REGION END #    //  SKALogger.test_SKALogger_decorators
```

Comment:
Same as above. Done by itself after regeneration. Have fixed it.

Reply:
Just noticed that the Protected regions for decorators were removed during regeneration (newer version of Pogo). There were also some other changes like, 

`def func(self) `

rather than 

`def func(self, tango_context)`

I had to add them manually later.

Reply:
Please use the "SKA" modified version of POGO @kmadisa provided via Slack - then you won't have do these things manually.  Here is the link again https://drive.google.com/file/d/1M6pPi4494tq7p1UlJTO1gDDtfYkUmfFh/view

Reply:
Okay, thanks.

Reply:
This test file still needs to be regenerated with POGO and updated, or at least restore **all** the blocks for decorator protection, like these:
```
# PROTECTED REGION ID(SKALogger.test_SKALogger_decorators) ENABLED START #
# PROTECTED REGION END #    //  SKALogger.test_SKALogger_decorators
```

Reply:
Okay. Will do it.